src/compiler/ast-node-conv.bas

```diff
@@ -492,22 +492,27 @@ function astNewCONV _
 
 		n->cast.convconst = ( symbCheckConstAssign( to_dtype, ldtype, to_subtype, l->subtype, , , wrnmsg ) = FALSE )
 
-		'' else check if const conversion
+		'' use the function pointer warning message?
+		if( wrnmsg <> 0 ) then
+			if( (options and AST_CONVOPT_DONTWARNFUNCPTR) = 0 ) then
```

Comment:
Doesn't the code here also handle non-function pointers? So I'm not fully sure which part ensures that the new option affects function pointers only.

Reply:
True, handles all pointers.  It's a hack: currently the warning message is only set by hSymbCheckConstAssignFuncPtr() which is only called for function pointer types.